backend/api/test_credit_trades.py

```diff
@@ -483,3 +483,216 @@ def test_delete(self):
+            initiator=test_organization_1,
+            respondent=test_organization_2,
+            type=credit_trade_type,
+            number_of_credits=1000,
+            fair_market_value_per_credit=1,
+            zero_reason=None,
+            trade_effective_date=datetime.datetime.
+            today().strftime('%Y-%m-%d'))
+
+        # the function shouldn't see this as it's a submitted transaction
+        # not involving the fuel supplier
+        submitted_credit_trade = CreditTrade.objects.create(
+            status=submitted_status,
+            initiator=test_organization_2,
+            respondent=test_organization_3,
+            type=credit_trade_type,
+            number_of_credits=1000,
+            fair_market_value_per_credit=1,
+            zero_reason=None,
+            trade_effective_date=datetime.datetime.
+            today().strftime('%Y-%m-%d'))
+
+        # the function should see this as it's a submitted transaction
+        # involving the fuel supplier
+        submitted_credit_trade_as_respondent = CreditTrade.objects.create(
+            status=submitted_status,
+            initiator=test_organization_2,
+            respondent=test_organization_1,
+            type=credit_trade_type,
+            number_of_credits=1000,
+            fair_market_value_per_credit=1,
+            zero_reason=None,
+            trade_effective_date=datetime.datetime.
+            today().strftime('%Y-%m-%d'))
+
+        # the function shouldn't see this as it's approved
+        approved_credit_trade = CreditTrade.objects.create(
+            status=approved_status,
+            initiator=test_organization_1,
+            respondent=test_organization_2,
+            type=credit_trade_type,
+            number_of_credits=1000,
+            fair_market_value_per_credit=1,
+            zero_reason=None,
+            trade_effective_date=datetime.datetime.
+            today().strftime('%Y-%m-%d'))
+
+        # the function should see this as it's completed
+        completed_credit_trade = CreditTrade.objects.create(
+            status=completed_status,
+            initiator=test_organization_1,
+            respondent=test_organization_2,
+            type=credit_trade_type,
+            number_of_credits=1000,
+            fair_market_value_per_credit=1,
+            zero_reason=None,
+            trade_effective_date=datetime.datetime.
+            today().strftime('%Y-%m-%d'))
+
+        credit_trades = CreditTradeService.get_organization_credit_trades(
+            test_organization_1
+        )
+
+        self.assertNotIn(draft_credit_trade, credit_trades)
+        self.assertIn(draft_credit_trade_from_fuel_supplier, credit_trades)
+        self.assertNotIn(submitted_credit_trade, credit_trades)
+        self.assertIn(submitted_credit_trade_as_respondent, credit_trades)
+        self.assertNotIn(approved_credit_trade, credit_trades)
+        self.assertIn(completed_credit_trade, credit_trades)
```

Comment:
Same comment as above. I don't think there's a good reason to bunch so many assertions in a single test. I think one should default to the "one assertion per test" approach unless there's a really good reason.